src/formContainer/FormContainer.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Form from '../form/Form.js';
+import './Styles.css'
+
+ function FormContainer() {
+
+    return(
+        <div className='formContainer'>
+            <Form/>
+        </div>
+    )
+}
+
+export default FormContainer
+
```

Comment:
Please rename component directory so it's consistent with component name (Capitalise first key)

Reply:
Corrected